src/auth_api/user.py

```diff
@@ -10,9 +10,8 @@ def create_or_get_user(
         state: AuthState
 ) -> DbUser:
     """
-        Creates or gets a user and an external user if they don't exist
+    Create or get a user and an external user if they don't exist.
```

Comment:
```suggestion
    Create or get a user and an external user if they don't exist.

```
Add new line between summary and extended summary.